### PR TITLE
New action to check minimal version of ruby

### PR DESCRIPTION
**Pull request by sebastianvarela**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [X] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [X] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [X] I've updated the documentation if necessary.

### Motivation and Context
In some cases, is necessary to ensure a minimal version of ruby. For this reason i made this action.

### Description
Check the ruby version to ensure compatibility with our Fastfile

### CODE REVIEW

**Comment by sebastianvarela** [comment]
I signed it!

File: fastlane/lib/fastlane/actions/ruby_version.rb
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/ruby_version.rb]: This should be your GitHub username 👍

File: fastlane/lib/fastlane/actions/ruby_version.rb
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/ruby_version.rb]: This sentence doesn't belong here, right?

**Review by KrauseFx — APPROVED** [review]
Hey @sebastianvarela, thanks for your contribution. Usually we don't accept new actions into fastlane right now, but this addition seems really useful. I'll let other fastlane core contributors chime in before we merge it 👍

File: fastlane/lib/fastlane/actions/ruby_version.rb
Comment (sebastianvarela) [inline fastlane/lib/fastlane/actions/ruby_version.rb]: @KrauseFx thanks! I change the author to my username 🤠

File: fastlane/lib/fastlane/actions/ruby_version.rb
Comment (sebastianvarela) [inline fastlane/lib/fastlane/actions/ruby_version.rb]: Done, I delete it. Does not contribute so much.

**Comment by ohayon** [comment]
This is great @sebastianvarela! I just added back the detail and we are good to 🚀 !

**Pull request merged**
